hacking/build_library/build_ansible/announce.py

```diff
@@ -1,5 +1,5 @@
 # coding: utf-8
-# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2020, Ansible Project
```

Comment:
I don't care but do note that the advice from Fontana is not too update the date here.  Sivel might have the page bookmarked that had that and other comments on legal boilerplate.

Reply:
@abadger Fair, I can pull that edit out. The guideline I'm familiar with was to bump it for "substantial" changes, but I don't know if this counts as that.